### PR TITLE
Block Factory: Nit UI Changes

### PR DESCRIPTION
**Pull request by quachtina96**
### All the "Block" buttons are grouped on the left and all "Block Library" buttons are grouped on right.
screen shot 2016-08-18 at 11 42 25 pm

### Helper text is now more detailed. 'Select From Library' rather than 'Select'. Block Selector fits better on page.
screen shot 2016-08-18 at 11 42 44 pm

screen shot 2016-08-18 at 11 43 14 pm

---
This change is [Reviewable](https://reviewable.io/reviews/google/blockly/570)

### CODE REVIEW

File: demos/blocklyfactory/index.html [hunk 1]
```diff
@@ -115,6 +120,7 @@
         <button id="exporterSubmitButton"> Export </button>
     </div>
     <div id="exportPreview">
+      <br>
```
Comment (NeilFraser) [inline demos/blocklyfactory/index.html]: Consider using a CSS rule for h3 instead of these br tags.

File: demos/blocklyfactory/index.html [hunk 2]
```diff
@@ -268,9 +274,9 @@
             <button id="removeBlockFromLibraryButton" title="Remove block from Block Library.">
               <span>Delete Block</span>
             </button>
-            <button id="clearBlockLibraryButton" title="Clear Block Library.">
-              <span>Clear Library</span>
-            </button>
+            <button id="createNewBlockButton" title="Create a new block.">
+              <span> Create Block</span>
+             </button>
```
Comment (NeilFraser) [inline demos/blocklyfactory/index.html]: There's an extra space here.

**Comment by quachtina96** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[demos/blocklyfactory/index.html, line 279 \[r2\]](https://reviewable.io:443/reviews/google/blockly/570#-KPYsOqJ0NqC53T1oSWL-r2-279:-KPYso-SKjtqk05-j11b:b-896fix) ([raw file](https://github.com/google/blockly/blob/01a6adbdaca2691e9b8aaf92311c3e0d5aff33ac/demos/blocklyfactory/index.html#L279)):*
<details><summary><i>Previously, NeilFraser (Neil Fraser) wrote…</i></summary>
> There's an extra space here.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly/570)*

**Comment by quachtina96** [comment]
@vicng @picklesrus @evd2014

---

Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly/570#-:-KPZPjNGnLwg96qGiE7n:b-c2xsk1)*

**Comment by evd2014** [comment]
Review status: 0 of 2 files reviewed at latest revision, 4 unresolved discussions.

---

*[demos/blocklyfactory/index.html, line 64 \[r3\]](https://reviewable.io:443/reviews/google/blockly/570#-KPn7jQfGppBWygE-IHB:-KPn7jQfGppBWygE-IHC:b-6a6km8) ([raw file](https://github.com/google/blockly/blob/18924edc699cdf6052a92eb72296f84967334e7a/demos/blocklyfactory/index.html#L64)):*
> ```HTML
>         <br>
>         <br>
>        Use the Export Settings form to download starter code for selected blocks.
> ```

Might make more sense to have the helper text for the Export Settings form in the Export settings column? You have plenty of room in that column, especially since the workspace factory stuff won't be in the exporter

---

*[demos/blocklyfactory/index.html, line 124 \[r3\]](https://reviewable.io:443/reviews/google/blockly/570#-KPn83AoRk02yzfHxpFj:-KPn83ApFyQR_UTOfm48:b9hsgk3) ([raw file](https://github.com/google/blockly/blob/18924edc699cdf6052a92eb72296f84967334e7a/demos/blocklyfactory/index.html#L124)):*
> ```HTML
>     <div id="exportPreview">
>       <br>
>       <h3>Preview Workspace: </h3>
> ```

Can remove exportPreview div now (I can do that in a later CL or you can do it here).

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly/570)*

**Comment by evd2014** [comment]
Review status: 0 of 2 files reviewed at latest revision, 4 unresolved discussions.

---

*[demos/blocklyfactory/index.html, line 124 \[r3\]](https://reviewable.io:443/reviews/google/blockly/570#-KPn83AoRk02yzfHxpFj:-KPn8OtwLa5iNz4kJonM:b-yjpv5d) ([raw file](https://github.com/google/blockly/blob/18924edc699cdf6052a92eb72296f84967334e7a/demos/blocklyfactory/index.html#L124)):*
<details><summary><i>Previously, evd2014 (Emma Dauterman) wrote…</i></summary>
> Can remove exportPreview div now (I can do that in a later CL or you can do it here).
</details>
Oops just checked the next CL and remembered how you're using the preview now. Disregard this comment. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly/570)*

**Comment by quachtina96** [comment]
Review status: 0 of 2 files reviewed at latest revision, 4 unresolved discussions.

---

*[demos/blocklyfactory/index.html, line 64 \[r3\]](https://reviewable.io:443/reviews/google/blockly/570#-KPn7jQfGppBWygE-IHB:-KPnLxS2pfAsxw41uRqP:bbmvif6) ([raw file](https://github.com/google/blockly/blob/18924edc699cdf6052a92eb72296f84967334e7a/demos/blocklyfactory/index.html#L64)):*
<details><summary><i>Previously, evd2014 (Emma Dauterman) wrote…</i></summary>
> Might make more sense to have the helper text for the Export Settings form in the Export settings column? You have plenty of room in that column, especially since the workspace factory stuff won't be in the exporter
</details>
Done.


---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly/570)*

**Comment by evd2014** [comment]
:lgtm:

---

Review status: 0 of 3 files reviewed at latest revision, 4 unresolved discussions.

---

*[demos/blocklyfactory/index.html, line 64 \[r3\]](https://reviewable.io:443/reviews/google/blockly/570#-KPn7jQfGppBWygE-IHB:-KPnSyq2Es3TL-16ItsS:b6x9tza) ([raw file](https://github.com/google/blockly/blob/18924edc699cdf6052a92eb72296f84967334e7a/demos/blocklyfactory/index.html#L64)):*
<details><summary><i>Previously, quachtina96 (Tina Quach) wrote…</i></summary>
> Done.
> 
> 
</details>
Nice! I like that much better!

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly/570#-:-KPnT3CMzgNG8SaJ2Q_W:bnfp4nl)*

**Comment by picklesrus** [comment]
Review status: 0 of 3 files reviewed at latest revision, 5 unresolved discussions.

---

*[demos/blocklyfactory/factory.css, line 212 \[r4\]](https://reviewable.io:443/reviews/google/blockly/570#-KPnTGPHPsOUGCdV7QY8:-KPnTGPHPsOUGCdV7QY9:b-v794dw) ([raw file](https://github.com/google/blockly/blob/29e0e75959d7bbcd5abbd3cc4768e11471ca08e2/demos/blocklyfactory/factory.css#L212)):*
> ```CSS
> 
> ::-webkit-scrollbar {
>     -webkit-appearance: none;
> ```

why are you hiding the scrollbars?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly/570)*

**Comment by vicng** [comment]
:lgtm:

---

Review status: 0 of 3 files reviewed at latest revision, 5 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly/570#-:-KPoLIdhKFR8hZGxo5Mb:bnfp4nl)*

**Comment by quachtina96** [comment]
Review status: 0 of 3 files reviewed at latest revision, 5 unresolved discussions.

---

*[demos/blocklyfactory/factory.css, line 212 \[r4\]](https://reviewable.io:443/reviews/google/blockly/570#-KPnTGPHPsOUGCdV7QY8:-KPojEsmZsvNj8CA771R:bevmv2t) ([raw file](https://github.com/google/blockly/blob/29e0e75959d7bbcd5abbd3cc4768e11471ca08e2/demos/blocklyfactory/factory.css#L212)):*
<details><summary><i>Previously, picklesrus wrote…</i></summary>
> why are you hiding the scrollbars?
</details>
This doesn't hide the scrollbars-- it makes them visible. I referenced this:

http://stackoverflow.com/questions/7492062/css-overflow-scroll-always-show-vertical-scroll-bar

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly/570)*

**Comment by picklesrus** [comment]
:lgtm:

---

Review status: 0 of 3 files reviewed at latest revision, 5 unresolved discussions.

---

*[demos/blocklyfactory/factory.css, line 212 \[r4\]](https://reviewable.io:443/reviews/google/blockly/570#-KPnTGPHPsOUGCdV7QY8:-KPokF3RTyTVnYHfMs_R:b-dowc4n) ([raw file](https://github.com/google/blockly/blob/29e0e75959d7bbcd5abbd3cc4768e11471ca08e2/demos/blocklyfactory/factory.css#L212)):*
<details><summary><i>Previously, quachtina96 (Tina Quach) wrote…</i></summary>
> This doesn't hide the scrollbars-- it makes them visible. I referenced this:
> 
> http://stackoverflow.com/questions/7492062/css-overflow-scroll-always-show-vertical-scroll-bar
</details>
Right, sorry. Completely read the code backwards. I've even read this stack overflow post before :)

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly/570#-:-KPonYurvrhlhFdqC3zQ:bnfp4nl)*

**Pull request merged**